translations/stripes-acq-components/en.json

```diff
@@ -501,6 +501,8 @@
   "organization": "Organization",
   "filter.organization.lookup": "Organization look-up",
   "filter.organization.lookupNoSupport": "Organization look-up is not supported",
+  "filter.user.lookup": "User look-up",
```

Comment:
@dltj Hi Peter, could this be translated please?

Reply:
Done in https://github.com/folio-org/stripes-acq-components/pull/234

Reply:
Thanks! :1st_place_medal: